pandas/_libs/tslibs/period.pyx

```diff
@@ -1270,6 +1270,36 @@ cdef class _Period(object):
 
     @property
     def days_in_month(self):
+        """
+        Get the total number of days in the month that this period falls on.
+
+        Returns
+        -------
+        int
+
+        See Also
+        --------
+        Period.daysinmonth : Returns the number of days in the month.
+        DatetimeIndex.daysinmonth : Return the number of days in the month.
+        calendar.monthrange : Return a tuple containing weekday
+            (0-6 ~ Mon-Sun) and number of days (28-31).
+
+        Examples
+        --------
+        >>> p= pd.Period('2018-2-17')
+        >>> p.days_in_month
+        28
+
+        >>> import datetime
+        >>> pd.Period(datetime.datetime.today(), freq= 'B').days_in_month
```

Comment:
Can you avoid `datetime.today()`? This won't pass doctests, aside from ~half the months in the year :)

Reply:
yeah sure !! Please pardon my naivety and this is a lifetime learning for sure. :)

Reply:
Should I just remove the second example because I couldn't think of any other way around of using this attribute different than the first example?

Reply:
Using `Period('2018-03-01').days_in_month` should be fine.